src/services/fs/dir_stream.rs

```diff
@@ -104,7 +114,12 @@ impl BlockingObjectPage for BlockingDirPager {
                 None => break,
             };
 
-            let path = build_rel_path(&self.root, &de.path().to_string_lossy());
+            dbg!(&self.root);
```

Comment:
Remove `dbg!`

Reply:
Done in ee705357979c02ee6d83913f24d567df80a065a4